coalib/processes/Processing.py

```diff
@@ -114,7 +114,10 @@ def autoapply_actions(results,
     """
 
     default_actions, invalid_actions = get_default_actions(section)
-
+    try:
+        flag = section['no_autoapply_warn']
+    except:
+        flag = False
```

Comment:
`flag` is a bad variable name. Also this can be simplified and you need to convert the value into a boolean, use something like: `no_autoapply_warn = bool(section.get('no_autoapply_warn', 'False'))`